invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/macros/detail.html

```diff
@@ -78,46 +78,38 @@ <h2>{{add_description.type.title_l10n }} <span class="text-muted language">{{ '(
 
 
 {% macro show_funding(funding) %}
-  <div class="ui middle divided aligned relaxed list" role="list">
     {% for fund in funding %}
       {{ _show_funding_item(fund, loop.index0) }}
     {% endfor %}
-  </div>
 {% endmacro %}
 
 {% macro _show_funding_item(item, index) %}
-<div class="item" role="listitem" id="funding-item-{{ index }}">
-  <div class="content">
-    {%- if item.award -%}
-      {%- if item.award.title.en -%}
-        <div class="header">
-          <span class="mr-5">
-            {{ item.award.title.en }}
+  {%- if item.award -%}
+    {%- if item.award.title.en -%}
```

Comment:
@jrcastro2 possible conflict with your PRs?

Reply:
Yes,this will conflict, we should use `title_l10n` as soon as the PRs are merged
```suggestion
    {%- if item.award.title_l10n -%}
```